src/editor/plugins.js

```diff
@@ -104,6 +107,23 @@ export function getPlaceholderPlugin({ text }) {
     }
   }
 
+  function createPlaceholderNode(text) {
+    let span = document.createElement('span');
+    span.innerHTML = `
+<style>
```

Comment:
Didn't realise it was invalid HTML - can't nest a `style` inside a `span`?

Tried `pointer-events`, doesn't seem to work - not on inline elements at least, as there's nothing to focus on it seems.

Absolutely positioned part - yeah figured that might be the case. At this point it's feeling a bit like what's the least buggy solution we can come up with right now. I can have another look at it though.